WcaOnRails/app/webpacker/lib/requests/routes.js.erb

```diff
@@ -88,6 +90,8 @@ export const adminImportResultsUrl = (competitionId) => `<%= CGI.unescape(Rails.
 
 export const adminCheckUploadedResults = (competitionId) => `<%= CGI.unescape(Rails.application.routes.url_helpers.competition_admin_upload_results_edit_path("${competitionId}"))%>`;
 
+export const delegatesApiUrl = `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_delegates_path)%>`;
```

Comment:
Is this used anywhere? I'm not sure if this is one of the URL that I've removed.

Reply:
Yes, this is used for loading the list of delegates for the delegate selection filtering in the overview page.